### PR TITLE
Show "account disabled" message in readable place when user logging in has been deactivated

### LINKED ISSUE #23432: "Your account is disabled" message overflows sign in modal
**Describe the bug**
When admin deactivates a user and that user tries to log in again, the message "Your account is disabled" is displayed. But it looks weird in the log in modal and it overflows it.

**Logs**
N/A

**To Reproduce**
1. As admin deactivate a user
2. Log out
3. Try to log in as deactivated user
4. See the error message crammed up next to the sign in button and slightly overflowing the modal

image

**Expected behavior**
The error message should follow the logical flow of the modal and be displayed below "Sign in" button.

**Information about your Metabase Installation:**
local dev, `master`, H2, Sample Database

**Severity**
extremely minor because it affects only deactivated users and it's a CSS/UI thing.

**Additional context**
I didn't check how far back this issue goes, but my guess is that it was always like that.

### PR DESCRIPTION
**Pull request by gusaiani**
Fixes #23432

### How to Test

1. As admin, deactivate a user
2. Log out
3. Try to log in as deactivated user

Message should no longer be crammed up to the right of the form.

### After

image

### CODE REVIEW

File: frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+
+interface CustomFormFooterStyledProps {
+  shouldReverse?: boolean;
+}
+export const CustomFormFooterStyled = styled.div<CustomFormFooterStyledProps>`
+  display: flex;
+  flex-align: center;
```
Comment (WiNloSt) [inline frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx:8]: This is a typo, there's no such CSS property. This cause the UI button to go full width.
```suggestion
  align-items: center;
```
#### Before
image

#### After
image

**Review by WiNloSt — CHANGES_REQUESTED** [review]
After fixing the CSS property typo this PR should be ready to merge 👍

**Comment by gusaiani** [comment]
@WiNloSt Thanks for catching it, updated <3

**Comment by WiNloSt** [comment]
@gusaiani Did you perhaps not push the update? I still do not see the supposed updated code 😅 
image

**Comment by gusaiani** [comment]
@WiNloSt Yes, my ext keyboard actually died a few mins ago, true story. I use `p Enter` for pushing, `p` never happened.

File: frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+
+interface CustomFormFooterStyledProps {
+  shouldReverse?: boolean;
+}
+export const CustomFormFooterStyled = styled.div<CustomFormFooterStyledProps>`
+  display: flex;
+  align-items: center;
```
Comment (WiNloSt) [inline frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx:8]: Well... correctly the property name seems to make some forms have the button right in the middle as we're now changing the main flex axis to the vertical axis, so the `align-items` which aligns the flex items along the secondary axis (now horizontal axis), now place the item in the middle.

image

Could you at least make sure both:
- Slack form http://localhost:3000/admin/settings/slack and
- Login form

still works as expected?

Comment (WiNloSt) [reply frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx:8]: FYI @gusaiani

**Review by WiNloSt — APPROVED** [review]
Looks good now 👍

**Comment by gusaiani** [comment]
Thank you very much for the review, @WiNloSt

**Pull request merged**